pkg/provisioner/ironic/ironic.go

```diff
@@ -353,6 +353,44 @@ func (p *ironicProvisioner) findExistingHost() (ironicNode *nodes.Node, err erro
 
 }
 
+func (p *ironicProvisioner) updateAutomatedClean(ironicNode *nodes.Node) (err error) {
+	var updates nodes.UpdateOpts
+
+	if p.host.Spec.AutomatedCleaningMode == "Disabled" {
```

Comment:
Maybe this logic can be collapsed so that we're only creating the `updates` instance in one place and the conditional only sets the `Value` argument?

Reply:
That's a good point. Fixed it.